Sources/Deboogger.swift

```diff
@@ -86,40 +113,61 @@ public final class Deboogger {
         NotificationCenter.default.post(name: .DebooggerWillHide, object: nil)
 
         pluginViewController?.dismiss(animated: true, completion: { [unowned self] in
-            self.assistiveButtonWindow.isHidden = false
+            self.isShowing = false
+            self.assistiveButtonWindow.isHidden = !self.shouldShowAssistiveButton
             self.rootViewController?.endAppearanceTransition()
             NotificationCenter.default.post(name: .DebooggerDidHide, object: nil)
         })
     }
 
-    // MARK: - Helpers
+    public func show() {
+        if isShowing {
+            close()
```

Comment:
It's strange behavior when you call `show` and it's closed. It's better to rename this function